app/components/schools/participants/status.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Schools
+  module Participants
+    class Status < BaseComponent
+      def initialize(participant_profile:)
+        @participant_profile = participant_profile
+      end
+
+    private
+
+      attr_reader :participant_profile
+
+      def heading
+        t :header, scope: translation_scope
+      end
+
+      def content
+        Array.wrap(t(:content, scope: translation_scope))
+      end
+
+      def translation_scope
+        @translation_scope ||= "schools.participants.status.#{profile_status}"
+      end
+
+      def profile_status
+        if (eligibility = participant_profile.ecf_participant_eligibility)
+          return :ineligible if eligibility.ineligible_status?
+
```

Comment:
We needs to update the definition here for ineligibility, we aren't planning on relying on the ineligible status yet. Ineligibility for this release will be `ineligibility: { status: :manual_check, reason: %i[active_flags previous_induction] }) }`

Reply:
Well apparently we are now going to have ineligible, but I think we will still need to include the manual check / active_flags/previous_induction participants